source/utilities.tex

```diff
@@ -5896,7 +5892,6 @@
 \begin{itemdescr}
 \pnum
 \effects
-Constructs an object of class \tcode{bitset<N>} as if by:
```

Comment:
The "as if by:" should be kept (properly capitalized).

Reply:
Fixed.